bin/check_configs.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Checks the validity of RuboCop config files in `conf/` directory (excluding the base config).
+
+CONFIGS=(
+  "conf/rubocop_gem.yml"
```

Comment:
`rubocop.yml` was omitted from this list since it gets inherited by both these configs:

https://github.com/ezcater/ezcater_rubocop/blob/24b52dac3b57eef05ad76d4e42803273fa690829/conf/rubocop_gem.yml#L1-L2